src/app/header/header.component.scss

```diff
@@ -37,7 +39,7 @@
     .nav-mob-scroll{
         overflow-x: scroll !important;
         .nav-list-mob-scroll{
-            width: 150%;
+            width: 170%;
```

Comment:
I believe this feature is automatically built into either Bootstrap or PatternFly. A width over 100% should be avoided for browser rendering and accessibility.